kdc/krb5tgs.c

```diff
@@ -1777,7 +1777,27 @@ tgs_build_reply(astgs_request_t priv,
 		goto out;
 	    }
 
+	    /* Verify the PAC of the TGT. */
+	    ret = check_PAC(context, config, user2user_princ, NULL,
```

Comment:
Does need to be behind require_pac as well?

Reply:
`check_PAC()` (now `_kdc_check_pac()`) does that.

Reply:
The PAC will still always be required for user-to-user due to the `user2user_pac == NULL || !user2user_kdc_issued` check. For the FAST armor ticket check I changed a similar check to `(r->config->require_pac && !mspac) || (mspac && !ad_kdc_issued)`.